platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java

```diff
@@ -138,14 +161,81 @@ public void doImport(ImportMigrationContext context) {
         .stream()
         .map(context::getEntry)
         .forEach(me -> me.restore(false));
+    importKeystores(context);
+    importServiceWrapperFiles(context);
+  }
+
+  @Override
+  public void doVersionUpgradeImport(ImportMigrationContext context, String migratableVersion) {
+    if (Float.parseFloat(migratableVersion) > Float.parseFloat(getVersion())) {
+      context
+          .getReport()
+          .record(
+              new MigrationException(
+                  IMPORT_UNSUPPORTED_MIGRATABLE_VERSION_ERROR,
+                  migratableVersion,
+                  getId(),
+                  getVersion()));
+      return;
+    }
+
+    LOGGER.debug("Upgrading system properties...");
+    upgradeSystemProperties(context);
+
+    LOGGER.debug("Importing remaining system files...");
+    PlatformMigratable.UPGRADEABLE_SYSTEM_PATHS
+        .stream()
+        .map(context::getEntry)
+        .forEach(me -> me.restore(false));
+    importKeystores(context);
+    importServiceWrapperFiles(context);
+  }
+
+  private void upgradeSystemProperties(ImportMigrationContext context) {
```

Comment:
Could this be changed to `ugradeCustomSystemProperties` to go along with the updated file path name? 😄

Reply:
Yup 👍

Reply:
Fixed :)